helix-rest/src/test/java/org/apache/helix/rest/server/TestClusterAccessor.java

```diff
@@ -125,6 +126,93 @@ public void testGetClusterTopology() {
   }
 
   @Test(dependsOnMethods = "testGetClusterTopology")
+  public void testGetClusterTopologyAndFaultZoneMap() throws IOException {
```

Comment:
It'd be more complete if the bad response is also tested. We don't always have good response. Eg. the cluster is not setup -> Error 400 bad request.